src/OrchardCore.Modules/OrchardCore.PublishLater/Startup.cs

```diff
@@ -0,0 +1,37 @@
+using Fluid;
```

Comment:
Fluid should be go after `Microsoft.*` namespace

Reply:
Why? I'm using [this VS extension](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.ProductivityPowerPack2017) BTW that removes unneeded usings on save (small code cleanliness improvement), as well as sorts them alphabetically (not that the order would matter here much but if we want a specific order then that one is simple and automated, and you don't have to think about it, which is good for something that doesn't matter in the end :)).

Reply:
We have an `.editorconfig` file, the only rule is to put the System ones first, the other ones being ordered alphabetically.

    dotnet_sort_system_directives_first = true

So Fluid "should" be before Microsoft.* ;)

Reply:
Hmm, OK, the editorconfig is supposed to be applied, though no warning, message or anything is visible for this file. With all other namespaces the system ones are actually at the top so I don't know what's the difference with Fluid. Anyway, moved it to the top by hand.

Reply:
> the editorconfig is supposed to be applied

Under the VStudio editor you still need to right click and do `Remove and Sort Usings`.

Reply:
Yeah this is what the linked extension also does on save (it actually does alphabetic sorting just by default and follows whats's in the editorconfig otherwise). It seems Fluid is somehow breaking this feature.